### DISCUSSION TITLE
Override "Language for Feed Processing Reports" setting

### DISCUSSION THREAD

**Discussion by therubioftw** [amzn/selling-partner-api-models#3302] [General]
Shalom!

We have hundreds of clients connected to our company, and all of them have never set [this option](https://sellercentral.amazon.com/sw/AccountInfo/FeedProcessingReport/step/Language) to anything other than the default English.

Yesterday, for the first time, we got a report in German - and it broke most of the automation we have set up.

Is there a way to override this setting, to make sure reports are always generated in English on our end?

**Comment by barshopen** [comment]
@therubioftw did you find a solution ?

**Comment by therubioftw** [comment]
Sadly now - this can't be anything but an oversight. It doesn't make any sense...
It only affects about 0.3% of our clients, but when a report comes in with headers in different languages, it basically creates a whole cascade of errors that takes a while to fix.
